samples/outlook/preview-apis-signature/work-with-client-signatures.yaml

```
@@ -0,0 +1,115 @@
+id: outlook-signature-work-with-client-signatures
+name: Work with Client Signatures
```

I think these APIs only work apply to compose mode, right? If so, then name should be something like:
```suggestion
name: Work with Client Signatures (Compose)
```